### PR TITLE
Further emphasises the importance of round ID in issue template

### PR DESCRIPTION
**Pull request by Cyberboss**
Reee

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-[Directions]: # (INCLUDE THE ROUND ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a test merge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)
+[Directions]: # (
+
+
+INCLUDE THE ROUND ID 
+
+
+from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a test merge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)
```
Comment (nfreader) [inline .github/ISSUE_TEMPLATE.md]: URL is now https://atlantaned.space/statbus/round.php

**Comment by Tlaltecuhtli** [comment]
ic issue

**Comment by coiax** [comment]
Put the round ID in the Show Server Revision text blob.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Fix the URL, please.

**Pull request merged**